lib/matplotlib/colors.py

```diff
@@ -2066,3 +2074,83 @@ def from_levels_and_colors(levels, colors, extend='neither'):
 
     norm = BoundaryNorm(levels, ncolors=n_data_colors)
     return cmap, norm
+
+
+def darker(color, factor):
+    """
```

Comment:
These don't appear in the rendered docs, that I can see....

Reply:
https://26384-1385122-gh.circle-artifacts.com/0/home/circleci/project/doc/build/html/api/colors_api.html